code/src/index.css

```diff
@@ -1,13 +1,172 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: "Oxygen", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #000;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.popularList {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.popularList a {
+  width: 100%;
+  position: relative;
+  color: #fff;
+  text-decoration: none;
+  display: flex;
+}
+
+.popularList a img {
+  width: 100%;
+}
+
+.popularList .details {
```

Comment:
You could move this to tablet and computer query to not have hover effect on mobile since it's hard to navigate.